tests/mpi/distribute_flux_sparsity_pattern.cc

```diff
@@ -0,0 +1,323 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+/*
+ * Verify that the current commit fixes distributed flux sparsity
+ * patterns. This fixes a bug where (see the commit for more details) coupling
+ * between an active, locally owned cell and an active ghost cell may be
+ * erroneously skipped when building a sparsity pattern with flux terms.
```

Comment:
Please remove 'the current commit fixes'. After a while, this will be non-trivial and one would need to check the logs. I would prefer to simply state 'Verify that distributed flux sparsity patterns are built correctly. There used to be a bug where coupling ...'